lms/djangoapps/grades/models.py

```diff
@@ -529,7 +529,7 @@ class Meta(object):
     course_id = CourseKeyField(blank=False, max_length=255)
 
     # Information relating to the state of content when grade was calculated
-    course_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=False)
+    course_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=True, null=True)
```

Comment:
But it didn't allow it at the model level prior to this, right?: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/models.py